test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -76,10 +80,10 @@ end
 end
 
 @testset "Integer Linear tests" begin
-    MOIT.intlineartest(BRIDGED_OPTIMIZER, CONFIG, [
-        # TODO(odow): Indicator sets not supported.
-        "indicator1", "indicator2", "indicator3", "indicator4"
-    ])
+    # interval somehow needed for indicator tests
+    interval_optimizer = MOIB.LazyBridgeOptimizer(OPTIMIZER)
+    MOIB.add_bridge(interval_optimizer, MOIB.Constraint.SplitIntervalBridge{Float64})
```

Comment:
Remove the above three lines. We just need `BRIDGED_OPTIMIZER` below.

Reply:
`BRIDGED_OPTIMIZER` is using all bridges, including the bridge from indicator to SOS, the test could thus pass without testing the real implementation if an error is made (like `MOI.supports_constraint` returning the wrong type

Reply:
Okay, but then we need to test the indicator tests with `interval_optimizer`. Currently, it is un-used.

Reply:
totally correct, just fixed that